bindings/python/test/test_arrow.py

```diff
@@ -293,28 +293,30 @@ def _create_nested_data(self):
             for k, v in _TYPE_NORMALIZER_FACTORY.items()
             if k.__name__ not in ("ObjectId", "Decimal128")
         }
-        schema["nested"] = struct([field(a, b) for (a, b) in schema.items()])
-
+        schema["list"] = list_(list_(int32()))
```

Comment:
Why nest this list?

Reply:
Yeah, I think we should add a list that contains documents, and a list that contains other lists as new tests.